### PR TITLE
A few small update.

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

File: theano/tests/test_printing.py [hunk 1]
```diff
@@ -672,15 +672,15 @@ def test_scan_debugprint5():
     | | |ScalarFromTensor [id BK] ''
     | |   |Elemwise{sub,no_inplace} [id C] ''
     | |Subtensor{::int64} [id BL] ''
-    | | |IncSubtensor{Inc;int64::} [id BM] ''
+    | | |IncSubtensor{Set;int64::} [id BM] ''
```
Comment (lamblin) [inline theano/tests/test_printing.py]: You will probably need to revert that one as well.

**Comment by nouiz** [comment]
I removed the commit that changed tests related to the reverted change.

File: doc/library/index.txt [hunk 2]
```diff
@@ -27,6 +27,7 @@ Types and Ops that you can use to build and compile expression graphs.
    sparse/sandbox
    tensor/index
    typed_list
+   tests
```
Comment (lamblin) [inline doc/library/index.txt]: It gives the following:
```
Warning, treated as error:
/home/jenkins-cpu1/workspace/Theano PR multijob/doc/library/index.txt:12: WARNING: toctree contains reference to nonexisting document u'library/tests'
```
I don't see any file named `tests.txt` in `doc/library` in the repo. Do you have it locally? Should you commit it?

**Comment by nouiz** [comment]
I added the missing file.

**Comment by nouiz** [comment]
close/reopen to retrigger travis. I don't understand why this PR can cause the travis problem.

**Pull request merged**